### ISSUE TITLE
code example Structured data learning with TabTransformer is worse than simple baselines

### ISSUE THREAD

**Issue by zenogantner** [keras-team/keras-io#1111]
[This blog post](https://medium.com/@tunguz/about-those-transformers-for-tabular-data-116c13c36a5c) criticizes [this example](https://keras.io/examples/structured_data/tabtransformer/) for failing to achieve the performance of logistic regression (84.5% vs. 85.3% accuracy) and the sklearn MLP (85.4%).

**Comment by sachinprasadhs** [comment]
Hi, 

Thanks for reporting the issue.

The main idea behind the published tutorials on Keras is to demonstrate the usage of APIs and workflow to build any model with the simple examples and sometimes with less number of epochs by considering the system computation level in normal scenarios.
We don't aim to prove any benchmark accuracy, hope this helps. Thanks!
